sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -205,15 +209,21 @@ def _check_db_exists(self):
                 self.existing_databases = {}
                 cursor.execute(DATABASE_EXISTS_QUERY)
                 for row in cursor:
-                    self.existing_databases[row.name] = True
+                    self.existing_databases[row.name.lower()] = 'CI' in row.collation_name, row.name
```

Comment:
(Nit) Took me a bit to realize 'CI' stood for 'case insensitive' and what the `collation_name` was, so perhaps an intermediate variable to help out readers?

```suggestion
                    case_insensitive = 'CI' in row.collation_name
                    self.existing_databases[row.name.lower()] = case_insensitive, row.name
```

Reply:
Nice improvement!